lnbits/core/views/api.py

```diff
@@ -196,25 +196,27 @@ class CreateInvoiceData(BaseModel):
 
 
 async def api_payments_create_invoice(data: CreateInvoiceData, wallet: Wallet):
+    description_hash = None
```

Comment:
- previously the default values were `b""` and not `None`

Reply:
Agree with @motorina0, let's not switcherydoo the types like Python programmers

Reply:
done